packages/components/docs/localDev.md

```diff
@@ -172,14 +175,20 @@ merge back to develop.
 Steps for package version numbering during feature branch development:
 1. Create your feature branch off of develop, i.e. fb_feature_1, and add your changes.
 1. When you are ready to push an alpha version up to Artifactory so you can test it in your application and
-on TeamCity, run the `yarn publish` command from the @labkey/components package root directory. This command
-will then prompt you for a new alpha package version. Ex. if adding a new feature and the current version is `0.1.0`,
-you would use `0.2.0-fb-feature-1.1`.
+on TeamCity, edit the version of the @labkey/components package in `package.json`.
+1. Use an alpha version number for the package of the form `X.Y.Z-fb-my-branch-name.0`, where `X.Y.Z` is your best guess at the
+next [SemVer](https://semver.org/) version that will include your changes, `fb-my-branch-name` is the name of your
+feature branch with underscores replaced by hyphens and the `.0` is just a starting point for the prerelease versioning.
+This `.0` version will be incremented with each alpha package version you want to publish and test of the module.
+1. Once you have modified the version in `package.json` run `npm install --legacy-peer-deps` to increment the version in the `package-lock.json`.
+1. Commit these changes using a message like `@labkey/components@X.Y.Z-fb-my-branch-name.0`.
+1. Run the `npm publish` command from the @labkey/components package root directory. This command will
```

Comment:
I did like that yarn did these things in a single step / command. This will take some getting used to but the old habit will likely break quickly.